src/core/eucovidcert.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_monitor_scheduled_query_rules_alert" "eucovidcert_availability_getcertificate" {
+  name                = format("[%s] %s %s", upper("eucovidcert"), "availability", "getcertificate")
+  resource_group_name = azurerm_resource_group.rg_external.name
+  location            = azurerm_resource_group.rg_external.location
+
+  action {
+    action_group = [azurerm_monitor_action_group.email.id, azurerm_monitor_action_group.slack.id]
+  }
+
+  data_source_id = data.azurerm_log_analytics_workspace.monitor_rg.id
+  # todo when will send logs to sec workspace
+  # data_source_id = var.env_short == "p" ? data.azurerm_key_vault_secret.sec_workspace_id[0].value : data.azurerm_log_analytics_workspace.monitor_rg.id
+  description             = "Availability for Get Certificate is less than or equal to 99%, check ${module.app_gw.name}"
+  enabled                 = var.eucovidcert_alerts_enabled
+  auto_mitigation_enabled = true
+
+  query = <<-QUERY
+    AzureDiagnostics
+    | where Resource == "IO-P-AG-APPGATEWAY" or Resource == "${upper(module.app_gw.name)}"
```

Comment:
Why `or`? I think we should just `Resource == "${upper(module.app_gw.name)}"`

Reply:
at the moment "IO-P-AG-APPGATEWAY" is the real appgateway, after migration complete we can remove "IO-P-AG-APPGATEWAY" from query.
In this way query will work during the switch